src/cross_section/src/cross_section.cpp

```diff
@@ -560,6 +612,42 @@ CrossSection CrossSection::Offset(double delta, JoinType jointype,
   return CrossSection(ps);
 }
 
+CrossSection CrossSection::Hull(
+    const std::vector<CrossSection>& crossSections) {
+  int n = 0;
+  for (auto cs : crossSections) n += cs.NumVert();
+  SimplePolygon pts;
+  pts.reserve(n);
+  for (auto cs : crossSections) {
+    auto paths = cs.GetPaths();
+    for (auto path : paths) {
+      for (auto p : path) {
+        pts.push_back(v2_of_pd(p));
+      }
+    }
+  }
+  return CrossSection(C2::PathsD{hull(pts)});
+}
+
+CrossSection CrossSection::Hull() const {
+  return Hull(std::vector<CrossSection>{*this});
+}
+
+CrossSection CrossSection::Hull(const SimplePolygon poly) {
+  auto pts = poly;  // hull will sort inplace, so we copy here
```

Comment:
Ah right, I'll drop the line then :+1:. Wasn't thinking of that.

Reply:
Oh right, the reason why I did it is that I actually get an error because the parameter is const and I'm passing it to the internal `hull` which is not `const` (because it needs to sort).

Reply:
No need to call it const when it's not taken by reference :).